trace-utils/src/trace_utils.rs

```diff
@@ -42,14 +42,14 @@ pub async fn get_traces_from_request_body(
 
 // Tags gathered from a trace's root span
 #[derive(Default)]
-struct RootSpanTags<'a> {
+pub struct RootSpanTags<'a> {
     pub env: &'a str,
     pub app_version: &'a str,
     pub hostname: &'a str,
     pub runtime_id: &'a str,
 }
 
-fn construct_trace_chunk(trace: Vec<pb::Span>) -> pb::TraceChunk {
+pub fn construct_trace_chunk(trace: Vec<pb::Span>) -> pb::TraceChunk {
```

Comment:
If we're making things public we should probably include doc comments.

Reply:
I got it wrong, it's used only inside the crate so I'm changing to `pub(crate)`.